pom.xml

```diff
@@ -1300,7 +1300,12 @@
             <dependency>
                 <groupId>commons-pool</groupId>
                 <artifactId>commons-pool</artifactId>
-                <version>1.6</version>
```

Comment:
let's just set it back to 1.6, as it was working that way and we can avoid another CQ.